arangod/Utils/SupportInfoBuilder.cpp

```diff
@@ -668,10 +668,10 @@ void SupportInfoBuilder::buildDbServerDataStoredInfo(
 
             auto flags = Index::makeFlags(Index::Serialize::Estimates,
                                           Index::Serialize::Figures);
-            constexpr std::array<std::string_view, 13> idxTypes = {
+            static constexpr std::array<std::string_view, 14> idxTypes = {
                 "edge",       "geo",      "hash",   "fulltext", "inverted",
                 "persistent", "skiplist", "ttl",    "mdi",      "mdi-prefixed",
-                "iresearch",  "primary",  "unknown"};
+                "iresearch",  "primary",  "vector", "unknown"};
```

Comment:
For telemetry schema consistency, we need to always report the vector index here, regardless of whether it is used or not.